### ISSUE TITLE
Repeatedly getting the same error

### ISSUE THREAD

**Issue by BigAlGit** [NecronomiconCoding/NecroBot#1781]
Screenshot of error is attached.  I get it every few minutes and then it restarts (shows the top pokemon and continues searching).  I am using a GPX file, the GPX file is in the main folder.  It stops at Pokestops along the route but seems to pop up the error randomly.

![error](https://cloud.githubusercontent.com/assets/20688149/17297971/00599adc-57d6-11e6-832a-9fd7d3f6b65c.PNG)

I can upload any settings or files requested.

Thanks!

**Comment by Andrerm124** [comment]
It SEEMS like it's an issue with the file already open by another process when it's trying to be accessed. It could be that the file every now and again is still open in another thread.

Are you using multiple bots at once? This could theoretically trigger this error.

**Comment by BigAlGit** [comment]
Well with your idea, I found out that it was due to the LastPos repeatedly updating.  I have the program set to run in my Dropbox folder so I can run it on different systems without issue.  I can't find any way to disable LastPos reporting, so I think I just have to manually copy the config and maps over when I want to save them.

I had it set to not sync the log folder since that updated a lot and it was great.  Unfortunately, the LastPos is in the config folder, which is with stuff that I want to backup.

Thanks for the help!
